cdc/log.cc

```diff
@@ -49,6 +49,7 @@
 #include "concrete_types.hh"
 #include "types/listlike_partial_deserializing_iterator.hh"
 #include "tracing/trace_state.hh"
+#include "stats.hh"
```

Comment:
I'm commenting here because I can't on the commit message:
s/cdc_service it is also going modify it/cdc_service it is also going to modify it/